### ISSUE TITLE
[BUG] Event created from app for "Group members only" does not appear on the admin website

### ISSUE THREAD

**Issue by vburgett** [rokwire/illinois-app#3934]
Today I created the event "group members only" from the app to publish in the Group "Would You Rather . . ."  I could see the event in the Group as desired. I could see the event under All Events, as _not_ desired. I could not see or access the event from the admin website, as expected.

v5.1.19

![Screenshot_20240202-094004](https://github.com/rokwire/illinois-app/assets/1044905/e1ce2cb3-6640-4be0-b8cc-3c397f558a51)

**Comment by dobromirdobrev** [comment]
This is a backend issue: https://github.com/rokwire/calendar-building-block/issues/419

**Comment by vburgett** [comment]
Both Group-Members Only and Uploaded Guest List Only events that are created in the app do not show up on the admin website. This is true in 5.1.29.

**Comment by vburgett** [comment]
Still the case with 5.1.34.

**Comment by dobromirdobrev** [comment]
Hi @vburgett ,

The events marked as "Group Members Only" or "Uploaded Guest List Only" should be visible in the Admin website for the latest version of the admin app (currently 5.1.2). 
Could you please double check and let us know if it works for you?

As regards to the "issue" that "Group Members Only" events are visible in "All Events" section of the UIUC app - these events will be visible only фор the creator of the event and this is expected behaviour. There are technical limitations for that.

So, to summarise:
Please, re-test this issue and if you still observe any problem, describe it and let us know what should be the expected behaviour. Thanks.

**Comment by vburgett** [comment]
I created an event from the app to publish in a Group as a Group Member Only event. I can see the event in the Group, under All Events, and from the admin website, all as expected. (I still have questions about a Group Member Only event showing up under All Events for the group admin but this is the agreed upon understanding right now.)
